compiler-rt/CMakeLists.txt

```diff
@@ -375,6 +375,25 @@ endif()
 if(MSVC)
   # FIXME: In fact, sanitizers should support both /MT and /MD, see PR20214.
   set(CMAKE_MSVC_RUNTIME_LIBRARY MultiThreaded)
```

Comment:
Let me ask a stupid question: Why is this even set? The VS sanitizers come in all flavors and are basically derived from llvm as far as I know

Reply:
Because Microsoft haven’t yet upstreamed their changes where they make it possible to build the sanitizers with any CRT. The code here, in its current state, only support this one setting - allegedly.